CONTRIBUTING.md

```diff
@@ -1,227 +1,150 @@
 
-We will do our best to keep the master branch as clean as possible, with tests passing all the times. However, it can happen that the master branch moves faster than the release cycle. To ensure you have the latest stable version, please refer to the [release on npm](https://www.npmjs.com/package/strapi).
+## Bugs
 
-If you send a pull request, please do it against the `master` branch. We are developing upcoming versions separately to ensure non-breaking changes from master to the latest stable major version.
+We are using [GitHub Issues](https://github.com/strapi/strapi/issues) to manage our public bugs. We keep a close eye on this so before filling a new issue, try to make sure the problem does not already exist.
 
-***
+## Get in Touch
 
-## Setup Development Environment
-To facilitate the contribution, we drastically reduce the amount of commands necessary to install the entire development environment. First of all, you need to check if you're using the [required versions of Node.js and npm](https://strapi.io/documentation/3.x.x/getting-started/install-requirements.html)
+- [Slack](https://slack.strapi.io/)
+- [Spectrum](https://spectrum.chat/strapi?tab=posts)
 
-**Note: Fish shell users** - due to the way fish shell deals with symlinks, the following steps will not work.
+---
 
-Then, please follow the instructions below:
+## Before Submitting a Pull Request
 
-#### 1. ▪️ Fork the repository
+The core team will review your pull request and will either merge it, request changes to it, or close it.
 
-[Go to the repository](https://github.com/strapi/strapi) and fork it to your own GitHub account.
+**Before submitting your pull request** make sure the following requirements are fulfilled:
 
-#### 2. 💿 Clone from your repository
+- Fork the repository and create your branch from `master`.
+- Run `yarn setup` in the repository root.
+- If you’ve fixed a bug or added code that should be tested, add tests and link the corresponding issue in either your commit or your PR!
+- Ensure the test suite passes:
+  - `yarn test:unit`
+  - `yarn test:front`
+- Make sure your code lints (`yarn lint`).
 
-```bash
-git clone git@github.com:YOUR_USERNAME/strapi.git
-```
+## Contribution Prerequisites
 
-#### 3. ⏳ Installation
+- You have [Node](https://nodejs.org/en/) at v10.x.x only and [Yarn](https://yarnpkg.com/en/) at v1.2.0+.
+- You are familiar with Git.
 
-Go to the root of the repository.
-```bash
-cd strapi
-```
+## Development Workflow
 
-**Two setup are available... with or without the front-end builds.**
+To facilitate the contribution, we drastically reduce the amount of commands necessary to install the entire development environment. First of all, you need to check if you're using the [required versions of Node.js and npm](https://strapi.io/documentation/3.x.x/getting-started/install-requirements.html)
```

Comment:
...we **have** drastically **reduced** the amount of commands necessary to install the entire development environment.